lib/generators/decidim/templates/invisible_captcha.rb

```diff
@@ -0,0 +1,9 @@
+InvisibleCaptcha.setup do |config|
+  config.honeypots           << 'another_fake_attribute'
```

Comment:
I'm not sure of what this does, it is not rendering anything similar inside the html.